project.clj

```diff
@@ -7,7 +7,9 @@
   :dependencies [[me.raynes/fs "1.4.6" :exclusions [org.clojure/clojure]]
                  [me.raynes/conch "0.8.0"]
                  [clj-time "0.6.0"]
+                 [cheshire "5.7.0"]
```

Comment:
There's a newer version out, `5.7.1`.  I'd consider using that unless there's a reason to be on this version.  It's likely that 5.7.0 *was* the latest when this work initially began.